api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -433,8 +433,28 @@ type NodeAgentConfig struct {
 	// Embedding NodeAgentConfigMapSettings
 	// +optional
 	NodeAgentConfigMapSettings `json:",inline"`
+	// Embedding KopiaRepoOptions
+	// +optional
+	KopiaRepoOptions `json:",inline"`
+}
+
+type KopiaRepoOptions struct {
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	CacheLimitMB *int64 `json:"cacheLimitMB,omitempty"`
+	// +kubebuilder:validation:Enum=normalGC;fastGC;eagerGC
```

Comment:
I think description would be good here, I do not know what the available values mean